pliers/extractors/text.py

```diff
@@ -313,3 +335,38 @@ def _extract(self, stim):
         features = ['sentiment_' + k for k in scores.keys()]
         return ExtractorResult([list(scores.values())], stim, self,
                                features=features)
+
+class SpaCyExtractor(TextExtractor):
+    
+    def __init__(self, model='en_core_web_sm', features=None):
+        #verify_dependencies(['spacy'])
+        try: 
+            nlp = spacy.load('en_core_web_sm')
+        except ImportError:
```

Comment:
Rather than raising an exception if `spacy.load`  fails, you could first try downloading it via `spacy_download`. One way to do this is to have a second, nested try/except block. I.e., the logic is: first try to load the desired language model. If it fails, try to download it. If *that* fails, only then do we raise an `ImportError`.